indico/modules/events/registration/util.py

```diff
@@ -141,6 +163,34 @@ def create_registration(regform, data, invitation=None):
     return registration
 
 
+def modify_registration(registration, data, event, invitation=None):
+    regform = registration.registration_form
+    data_by_field = registration.data_by_field
+
+    for form_item in regform.active_fields:
+        if form_item.parent.is_manager_only:
```

Comment:
when a manager edits the form this will prevent him from changing things in manager-only sections